index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <h4 id="logo">My Logo</h4>
+            <a href="#" class="menu-icon"><img src="images/Union.png" alt="menu bar"></a>
+        </nav>
+    </header>
+    <section>
+        <h1>I'm Leonardo Glad to see you!</h1>
+        <h2>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h2>
+        <h3>LET'S CONNECT</h3>
```

Comment:
- [ ] This element should be a link tag.

Reply:
This issue was successfully addressed